index.html

```diff
@@ -17,7 +17,7 @@
     <div class="container-fluid">
       <ul class="nav nav-pills">
       <li role="presentation" class="active"><a href="#">Home</a></li>
-      <li role="presentation"><a href="#">Profile</a></li>
+      <li role="presentation"><a href="contact.html">Contact</a></li>
       <li role="presentation"><a href="#">Messages</a></li>
```

Comment:
you may want to remove any links you're not using and make sure that the ones you keep all have a path to follow instead of `href="#"`